app/templates/content/suppliers-guide-g-cloud.html

```diff
@@ -0,0 +1,248 @@
+{% from 'macros/_breadcrumb.html' import breadcrumb %}
+{% extends "_base_page.html" %}
+
+{% block breadcrumb %}
+  {{ breadcrumb(crumbs) }}
+{% endblock %}
+
+{% block main_content %}
+
+  <div id="page" class="html-publications-show">
+
+
+  <header class="publication-header" id="contents"><div class="inner-block floated-children">
+  <div class="headings">
+  <p class="document-type">Guidance</p>
+  <h1>G-Cloud suppliers' guide</h1>
+  <p>Updated <abbr class="published-at date" title="2014-09-30T12:35:08+00:00">30 September 2014</abbr></p>
+  </div>
+  <nav class="in-page-navigation"><h2>Contents</h2>
+  <ol>
+  <li><a href="#about-g-cloud">Digital Marketplace and G-Cloud</a></li>
+  <li><a href="#how-to-apply">How to apply</a></li>
+  <li><a href="#when-to-apply">When to apply</a></li>
+  <li><a href="#editing-existing-services">Editing existing services</a></li>
+  </ol></nav>
+  </div>
+  </header>
+  <div class="block publication-content">
+    <div class="inner-block floated-children">
+      <div class="govspeak">
+  <h2 id="about-g-cloud">
+  <span class="number">1. </span>Digital Marketplace and G-Cloud</h2>
+
+  <p>
+  Public sector organisations and arm’s length bodies can use the Digital Marketplace to buy cloud-based services, like web hosting, site analytics or document collaboration tools.
+  </p><p>
+  If you want to sell your cloud services on the Digital Marketplace, you need to submit your service details via the G-Cloud framework.
+  </p><p>
+  You can apply to submit your services to the G-Cloud framework when an Official Journal of the European Union (OJEU) is open. On average, an OJEU will be open for 6 weeks, every 6 to 9 months.
+  </p><p>
+  Once your submission has been processed, buyers will be able to find your service live on the Digital Marketplace.
+  </p>
+
+  <h2 id="how-to-apply">
+  <span class="number">2. </span>How to apply</h2>
+
+  <p>You must:</p>
+
+  <ol>
+  <li>Sign up to the Crown Commercial Service (CCS) eSourcing Portal.</li>
+  <li>Register your expression of interest on the CCS eSourcing Portal when the OJEU opens.</li>
+  <li>Answer the invitation to tender (ITT) questions on the CCS eSourcing Portal.</li>
+  <li>Submit company and service information to the Digital Marketplace using the service submission portal (SSP).</li>
+  <li>Wait for assurance checks to be made on your information.</li>
+  </li>
+  </ol>
+  <p>
+  Each step in this process is mandatory.
+  </p>
+
+  <figure class="image embedded"><div class="img"><img alt="G-Cloud application process" src="/assets/compressed/supplying-process.png"></div>
```

Comment:
This image isn't loading:
![screen shot 2015-05-13 at 10 51 03](https://cloud.githubusercontent.com/assets/6525554/7608011/009b4e28-f95e-11e4-8670-b1ea19e4e848.png)